model/metrics/system-metrics.yaml

```diff
@@ -34,7 +34,7 @@ groups:
               value: 'interrupt'
             - id: steal
               value: 'steal'
-        brief: "The state of the CPU"
+        brief: "The CPU state for this data point. A system's CPU SHOULD be characterized *either* by data points with no `state` labels, *or only* data points with `state` labels."
         examples: ["idle", "interrupt"]
```

Comment:
Should we expand the examples to match process metrics too?

idle, interrupt, io, etc?

Reply:
Based on https://github.com/open-telemetry/semantic-conventions/pull/681/files#r1507565207 from @braydonk ,  I think we cannot do it seamlessly unless we revisit the `system.cpu.state` and `process.cpu.state` in order to evaluate if we could/should unify them.

Reply:
Ok I see. Let's get this clarification going then, I think aligning the definition is the least we can do now.